src/pages/posts/s02e03.md

```diff
@@ -0,0 +1,34 @@
+---
+title: DevOps Party Games - Don't Call It A Comeback (EMEA)
+subtitle: Broadcast live at <a href = "https://www.timeanddate.com/worldclock/fixedtime.html?msg=DevOps+Party+Games&iso=20210209T20&p1=3903" target = "_blank">Tuesday 9 February 20:00 UTC+1</a> 
```

Comment:
Minor thing, but shouldn't it be "broadcast **on**" (not at?).

Reply:
Yeah, this one should be changed!